index.html

```diff
@@ -3404,6 +3397,13 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+        <p>
+The media type `application/vc+ld+json` does not imply any particular securing
+mechanism. It describes an object that is securable, but a securing mechanism
+needs to be applied to enable verification of integrity. This also applies to
+media types that allow for transformation into `application/vc+ld+json`. Do not
```

Comment:
```suggestion
mechanism.  Do not
```

Reply:
"It describes an object that is securable, but a securing mechanism needs to be applied to enable verification of integrity." This sentence hurts... and I think we are doing a better job of addressing it in: 

- https://github.com/w3c/vc-data-model/pull/1101

Reply:
I have reworked that sentence in an attempt to make it less painful in [1e350e3](https://github.com/w3c/vc-data-model/pull/1107/commits/1e350e3f328707fbb4ee16012ef4d3aff4a5e56a)